config/webpack-config.js

```diff
@@ -111,18 +121,21 @@ const appsConf = {
   plugins: [
     COMMON_CHUNK_CONFIG,
     COMMON_UGLIFY_CONFIG
-  ]
+  ],
+  ...STATS_CONFIG
 };
 
 const spanishConf = {
+  cache: true,
   module: COMMON_MODULE_CONFIG,
   mode: 'production',
   output: {
     filename: 'spanish.js'
   },
   plugins: [
     COMMON_UGLIFY_CONFIG
-  ]
+  ],
+  ...STATS_CONFIG
```

Comment:
Since this is the only webpack config where the `STATS_CONFIG` is used multiple times, you could modify the loop at the end of the file:

```js
let currConfig;
for ( const key in configExports ) {
  currConfig = configExports[key];
  currConfig.stats = STATS_CONFIG.stats; // Or use Object.assign(…)
  if ( NODE_ENV === 'development' ) {
    // eslint-disable-next-line guard-for-in
    Object.assign( currConfig, devConf );
  }
}
```

That would render the need for the turning on the experimental flag unnecessary.

Or alternatively, the object spread could be used for mixing in the dev config as well.

Reply:
I don't think we have to use object spread or Object.assign uniformly in all cases.  I think the code reads wells as it is.